# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/test/proxy_tunnel_socket_unittest.cpp
# Commit: 8196045d38b5
# Full Hash: 8196045d38b50e04313725215c36ba1618c50486
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Create an ipdl struct called WebrtcProxyConfig, to cut down on the arg count on lots of functions. r=mjf,mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45562
# ==============================================================================

diff -r 6e1eba5c88ad -r 8196045d38b5 media/mtransport/test/proxy_tunnel_socket_unittest.cpp
--- a/media/mtransport/test/proxy_tunnel_socket_unittest.cpp	Wed Sep 18 14:27:42 2019 +0000
+++ b/media/mtransport/test/proxy_tunnel_socket_unittest.cpp	Tue Sep 17 18:19:04 2019 +0000
@@ -11,7 +11,6 @@
 
 #include "mozilla/net/NeckoChannelParams.h"
 #include "nr_socket_tcp.h"
-#include "nr_socket_proxy_config.h"
 #include "WebrtcTCPSocketWrapper.h"
 
 #define GTEST_HAS_RTTI 0
@@ -35,12 +34,7 @@
         mConnected(false) {}
 
   void SetUp() override {
-    nsCString alpn = NS_LITERAL_CSTRING("webrtc");
-    std::shared_ptr<NrSocketProxyConfig> config;
-    config.reset(new NrSocketProxyConfig(0, alpn, net::LoadInfoArgs(),
-                                         NrSocketProxyConfig::kForceProxy));
-    // config is never used but must be non-null
-    mSProxy = new NrTcpSocket(config);
+    mSProxy = new NrTcpSocket(nullptr);
     int r = nr_socket_create_int((void*)mSProxy.get(), mSProxy->vtbl(),
                                  &nr_socket_);
     ASSERT_EQ(0, r);